llvm/include/llvm/Transforms/pvi3/pvi3.h

```diff
@@ -0,0 +1,16 @@
+#pragma once
+
+#define DEBUG_TYPE "pvi3"
```

Comment:
@PoliakovVI, please use such type of macro only in cpp files

Reply:
Sure, done